WordPress/src/main/java/org/wordpress/android/ui/people/PeopleListFragment.java

```diff
@@ -157,6 +157,15 @@ public int getItemCount() {
         }
 
         @Override
+        public long getItemId(int position) {
```

Comment:
I swear I had that! It didn't even work at first when I added it because I removed the `getItemId` while I was changing to `RecyclerView`. I guess I forgot to put it back in, so sorry, fixing now!